<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_new_list_name</class>
 <widget class="QDialog" name="add_new_list_name">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#add_new_list_name{
	background-image : url(:/images/first back_ground.png);
}</string>
  </property>
  <widget class="QLabel" name="LA_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;MRT_Yalda&quot;;
color :rgb(255, 255, 255) ;
</string>
   </property>
   <property name="text">
    <string>Title   :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_next">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MRT_Thameen DemiBold&quot;;
color: rgb(60, 60, 60); 
                                     background-color: #F8F9D7; 
                                     border: 1px solid rgb(60, 60, 60);
                                     border-radius: 11px;</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MRT_Thameen DemiBold&quot;;
color: rgb(60, 60, 60); 
                                     background-color: #FFFFFF; 
                                     border: 1px solid rgb(60, 60, 60);
                                     border-radius: 14px;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioB_white">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(255, 255, 255) ;
</string>
   </property>
   <property name="text">
    <string>white</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioB_red">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(255, 255, 255) ;
</string>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioB_cyan">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(255, 255, 255) ;
</string>
   </property>
   <property name="text">
    <string>cyan</string>
   </property>
  </widget>
  <widget class="QLabel" name="LA_color">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;MRT_Yalda&quot;;
color :rgb(255, 255, 255) ;
</string>
   </property>
   <property name="text">
    <string>color   :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MRT_Thameen DemiBold&quot;;
color: rgb(60, 60, 60); 
                                     background-color: #F8F9D7; 
                                     border: 1px solid rgb(60, 60, 60);
                                     border-radius: 14px;</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="LA_creat_new_list">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(255, 255, 255) ;
font: 75 16pt &quot;MRT_Yalda&quot;;</string>
   </property>
   <property name="text">
    <string>    creating new list</string>
   </property>
  </widget>
  <widget class="QLabel" name="Warniing_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MRT_Yalda&quot;;
color :rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
